lib/security/openssl_misc_utils.pm

```diff
@@ -49,7 +49,7 @@ sub has_default_openssl3 {
 
 sub install_openssl {
     zypper_call 'in openssl' unless is_transactional;
-    if (is_sle '>=15-SP6') {
+    if (is_sle('=15-SP6') || is_sle('=15-SP7') {
```

Comment:
```suggestion
    if (is_sle('=15-SP6') || is_sle('<16') {
```
Perhaps we leave the door open for a possible 15-SP7?

Reply:
LGTM